scripts/deploy_docker.sh

```diff
@@ -0,0 +1,12 @@
+docker stop -f $(docker ps |grep -v rally|awk '{print $1}'|grep -v CON)
+docker run -tid  --name ig1 -p 47500:47500 -p 47100:47100 -p 10800:10800 -it apacheignite/ignite:latest
+docker run --rm --net=host --name VMM -p 9005:9005 -v /tmp:/tmp -itd mm
```

Comment:
do we need "-itd VMM" or it doesn't matter actually?

Reply:
This script has the potential to replace https://github.com/futurewei-cloud/alcor/blob/master/scripts/deploy.sh.

Could you provide similar features of the current deploy.sh, for example, 
- add Ignite deployment with the latest xml you provided
- remove existing containers if conflicted with new deployment
- Ubuntu and MacOs support (MacOS is optional) you can do it later when not so busy

Reply:
should we first merge this then create separate issue for replace deploy.sh

Reply:
We could merge this PR first but these scripts can be used in a limited scenario (new env, first deployment). We are not far away from replacing deploy.sh therefore this is a good time for deploy.sh.

Reply:
we need that tid

Reply:
I would suggest to merge it first, since the replace work is not simple enough to be contained